app/Models/Worker.php

```diff
@@ -67,46 +69,54 @@ public function getFullDataAttribute() {
 				'label' => __('admin/workers.workplaces'),
 				'options' => Workplace::select('name', 'id')->get(),
 				'optionsLabel' => 'name',
-				'value' => $this->workplaces()->select('name','workplaces.id')->get(),
+				'value' => $this->workplaces()->select('name', 'workplaces.id')->get(),
 			], [
-				'name' => 'Supervisor',
+				'name' => 'supervisor',
```

Comment:
i use this field to filter stuff out in the SupervisorController, make sure that the filter is still happening